### DISCUSSION TITLE
File type support

### DISCUSSION THREAD

**Discussion by S0CH13** [rocksdanister/lively#272] [Q&A]
I wanted to ask if there are plans on giving lively support for project.json file from wallpaper engines work shop....

**Comment by rocksdanister** [comment]
That is not the direction I wish to take this project, moreover their scene format is probably copyrighted.
So, no currently there is no plans to directly support wpe formats.

Lively can auto generate preview and thumbnail, you can extract the file and just drop it into Lively window if its standard video/html file.
